### PR TITLE
users, lists and items can now be destroyed from the command line

### PR DESCRIPTION
**Pull request by blafri**
Hey Felix,
I finished the assignment for destroying users, lists and objects from the command line. Let me know if everything looks ok.

Thanks,
Blayne

### CODE REVIEW

File: app/controllers/api/lists_controller.rb [hunk 1]
```diff
@@ -14,14 +14,26 @@ def create
       end
     end
 
+    def destroy
+      if @list.destroy
```
Comment (felixclack) [inline app/controllers/api/lists_controller.rb]: This allows me to destroy any list, regardless of whether it belongs to me or not. Is that the intention?

File: app/controllers/api/users_controller.rb [hunk 2]
```diff
@@ -17,8 +17,22 @@ def create
       end
     end
 
+    def destroy
+      user = User.find(params[:id])
```
Comment (felixclack) [inline app/controllers/api/users_controller.rb]: This lets me delete any user. Again, is that intentional?

Comment (blafri) [inline app/controllers/api/users_controller.rb]: Hey Felix,
Yeh I know that any user would be able to destroy items and users. I was just following along with the checkpoints and they did not have any authorization in them. They have it as an extra credit user story and was just gonna do it when I reach there. But I will do it one time today before our meeting.

Comment (felixclack) [inline app/controllers/api/users_controller.rb]: Cool. I was raising it so that you were aware. Sounds like you've got it all in hand. 



—
Thanks,
Felix

**Comment by felixclack** [comment]
:+1:

**Pull request merged**